src/core/components.py

```diff
@@ -162,7 +162,7 @@ def get_base_component_dict(self):
         )
 
         if self.target:
-            component["target"]: self.target
+            component["target"] = self.target
```

Comment:
This can just be straight up removed

Reply:
We have this still in lots of places, we will remove them in one go.